README.md

```diff
@@ -2,152 +2,316 @@ Plop
 ======
 
 Micro-generator framework that makes it easy for an entire team to create files with a level of uniformity.
+![plop demo](http://i.imgur.com/penUFkr.gif)
+
+> [Documentation also available on plopjs.com](https://plopjs.com/documentation/)
 
+# Getting Started
 [![npm](https://img.shields.io/npm/dm/plop.svg)](https://www.npmjs.com/package/plop)
+&nbsp;
 [![npm](https://img.shields.io/npm/v/plop.svg)](https://www.npmjs.com/package/plop)
-[![plop on slack](https://plopjs-slack-ofhdlhmvql.now.sh/badge.svg)](https://plopjs-slack-ofhdlhmvql.now.sh/)
+&nbsp;
+[![plop on slack](https://plopjs-slack-rjjmojauhb.now.sh/badge.svg)](https://plopjs-slack-rjjmojauhb.now.sh/)
 
-![plop demo](http://i.imgur.com/penUFkr.gif)
+## What is Plop?
+Plop is a what I like to call a "micro-generator framework." Now, I call it that because it is a small tool that gives you a simple way to generate code or any other type of flat text files in a consistent way. You see, we all create structures and patterns in our code (routes, controllers, components, helpers, etc). These patterns change and improve over time so when you need to create a NEW *insert-name-of-pattern-here*, it's not always easy to locate the files in your codebase that represent the current "best practice." That's where plop saves you. With plop, you have your "best practice" method of creating any given pattern in CODE. Code that you can be easily run from the terminal by simply typing `plop`. Not only does this save you from hunting around in your codebase for the right files to copy, but it also turns "the right way" into "the easiest way" to make new files.
```

Comment:
Few typos I guess:

* _is what I like_ 
* _Code that you can easily run_ or _Code that can be easily run_

This intro is *great* by the way. I'm simply wondering if we might split this bloc into separate paragraphs or not, for readability 🤔 
But that something we could easily improve later, so let's move on 👌